ruby/basic_ruby/basic_enumerable_methods.md

```diff
@@ -304,8 +304,8 @@ votes = ["Bob's Dirty Burger Shack", "St. Mark's Bistro", "Bob's Dirty Burger Sh
 
 votes.reduce(Hash.new(0)) do |result, vote|
   result[vote] += 1
-  result
```

Comment:
This line implicitly returns the `result` hash from the do block to be used for the next iteration. I hope that makes sense. In fact your proposed changes will throw errors. I hope that makes sense, happy to answer any further questions.